app/views/boards/_reorder.haml

```diff
@@ -0,0 +1,51 @@
+- reset_cycle
+#reorder-sections-table
+  .content-header
+    = content_for :reorder_title
+    #loading.float-right.hidden= loading_tag # TODO duplicate id
+    #saveerror.float-right.hidden
+      - # TODO duplicate id
+      = image_tag "icons/exclamation.png", title: 'Error', class: 'vmid', alt: '!'
+      Error, please refresh
+    .saveconf.float-right.hidden
+      = image_tag "icons/accept.png", title: 'Saved', class: 'vmid', alt: ''
+      Saved
+  %ul.sortable.table-list
+    - sections.each do |section|
+      %li.padding-5.section-ordered{class: cycle('even', 'odd'), data: {id: section.id, type: section.class.to_s, order: section.section_order}}
+        .section-ordered-handle
+          = image_tag "icons/arrow_double.png", title: 'Reorder', class: 'disabled-arrow', alt: '↕'
+        = link_to section.name, section
+        .float-right
+          = image_tag "icons/arrow_up.png", class: "section-up disabled-arrow"
+          = image_tag "icons/arrow_down.png", class: "section-down disabled-arrow"
+          = link_to url_for(section) + "/edit" do # TODO
```

Comment:
What's TODO here? `edit_section_url(section)` or something?

Reply:
Problem is that it's either edit_board_section_url or edit_index_post_url; url_for will give the correct thing for the model but there's no equivalent for the edit url and appending "/edit" is not exactly Good Form

Reply:
use a helper and check the class?

Reply:
`url_for(section, action: 'edit')`?

Reply:
or even probably `link_to section, action: 'edit'`, since I think it defers to url_for directly

Reply:
If that doesn't work, it looks like `edit_polymorphic_url(section)` may? https://api.rubyonrails.org/classes/ActionDispatch/Routing/PolymorphicRoutes.html

Reply:
the action ones didn't work but edit_polymorphic did